lib/win32/certstore/store_base.rb

```diff
@@ -92,13 +92,8 @@ def cert_get(certificate_thumbprint, store_name:, store_location:)
         thumbprint = update_thumbprint(certificate_thumbprint)
         cert_pem = get_cert_pem(thumbprint, store_name: store_name, store_location: store_location)
         cert_pem = format_pem(cert_pem)
-        if cert_pem.empty?
-          raise ArgumentError, "Unable to retrieve the certificate"
-        end
 
-        unless cert_pem.empty?
-          build_openssl_obj(cert_pem)
-        end
+        cert_pem.empty? ? cert_pem : build_openssl_obj(cert_pem)
```

Comment:
I think this should return nil or cert_pem/object. This would also clean `.empty?` checks at other places.

Reply:
Can we get on Teams and talk about this? I feel like we have competing agendas and I want to make sure I am doing this correctly.